playwright/tests/util/helper.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+export async function waitForWebApp(url: string | undefined, timeout = 30000) {
+  if (!url) {
+    throw new Error("URL is not defined");
+  }
+
+  const startTime = Date.now();
+  while (true) {
```

Comment:
What will happen if the app never becomes ready? Will this fall back to the timeout of the GitHub Actions step?

Reply:
It is going to throw an error based on the timeout defined in the signature of the function: https://github.com/radius-project/samples/pull/978/files#diff-01141eafefa05f5a0809324840c2ed0088261d39f5fddd078b9be73baaac81a6R3.

Reply:
Perfect, thanks!!